### ISSUE TITLE
Using ESCA key to login by --ssh argument

### ISSUE THREAD

**Issue by a2d8a4v** [wp-cli/wp-cli#5100]
It seems that we can not use wp-cli to remote with ECSA key?

If we want to log in some instance like AWS ec2 or instance in Google Computer Engine, It is necessary  to create a SSH key first. And then login by command like this:
`ssh -i /path/to/key.pem user@ipaddress -p port`

but a little pity about using --ssh on wp-cli is that, It seems like we cannot allocate the path to key file for login using.

**Comment by schlessera** [comment]
Indeed, this is currently not possible. I looked into the `before_ssh` hook, but that is useless here, I don't see any way of injecting this additional argument with the current code.

However, you can provide per-host configuration of the identity file in your configuration file (usually `~/.ssh/config`): https://man.openbsd.org/ssh_config.5#IdentityFile

**Comment by a2d8a4v** [comment]
I understood. So maybe It could be the one new function in one day? Maybe add one more argument --ssh-key-path= or something else like it. I guess.

**Comment by schlessera** [comment]
If we would add another argument, it would be more along the lines of `-ssh-args` and accept any additional arguments to SSH.

**Comment by a2d8a4v** [comment]
Thank you very much! Hope the new function would come soon!

**Comment by swissspidy** [comment]
ssh-args flag was added in #6206
